src/main/resources/database/postgresql/1/patch_28.sql

```diff
@@ -1,4 +1,4 @@
-CREATE OR REPLACE VIEW repbot.truncated_reputation_offset AS
+CREATE OR REPLACE VIEW repbot_schema.truncated_reputation_offset AS
```

Comment:
Could this have created inconsitent schemas? Is it fine for instances that already ran this migration to not have it run again with this change?

Reply:
The default setting for the database is repbot_schema. The migration failed if that was changed,which was the reason I noticed it. Since no one complained I guess no one ever changed it.